src/Microsoft.Azure.WebJobs.Host/FunctionExecutedContext.cs

```diff
@@ -0,0 +1,35 @@
+﻿// FunctionExecutedContext.cs
+
+using System;
+using System.Collections.Generic;
+using Microsoft.Azure.WebJobs.Host.Executors;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.WebJobs.Host
+{
+    /// <summary>
+    /// The context for an executed function. This needs to be expanded on later.
```

Comment:
Let's get all the comments up to date in a somewhat finalized form, with no TODOs, etc.